charts/agent/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 This file documents all notable changes to the Sysdig Agent Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.0.3
```

Comment:
Let's call these minor changes, and actually bump the version to 1.1.0. Going forward I think it would be a good idea to to reserve patch version for bugfixes and other trivial changes

Reply:
makes sense